index.html

```diff
@@ -14,6 +16,87 @@
+      <header class="left-header">
+        <h1>Brandon</h1>
+      </header>
+      <section class="contact">
+        <h3>CONTACT</h3>
+        
+        <p>ufc.campbell@gmail.com<br>217-851-0771</p>
+        <div class="media-links">
+          <a href="#">Twitter</a>
+          <a href="#">LN</a>
+          <a href="#">Git</a>
+          <a href="#">FB</a>
+        </div>
+      </section>
+      <section class="education">
+        <h3>EDUCATION</h3>
+        <p class="text"><span class="bold">2009</span><br> School - Course [explenation]<br><span class="location">Location</span></p> 
+        <p class="text"><span class="bold">2010-2011</span><br>School - Course [explenation]<br><span class="location">Location</span></p> 
+        <p class="text"><span class="bold">2011-2014</span><br>School - Course [explenation]<br><span class="location">Location</span></p> 
+        <p class="text"><span class="bold">2015</span><br>School - Course [explenation]<br><span class="location">Location</span></p> 
+        <p class="text"><span class="bold">2018</span><br>School - Course [explenation]<br><span class="location">Location</span></p> 
+        
+      </section>
+      <section class="skills">
+        <h3>SKILLS</h3>
+        <ul class="skills__list">
+          <li class="text">Language 1</li>
+          <li class="text">Language 2</li>
+          <li class="text">Language 3</li>
+          <li class="text">Language 4</li>
+          <li class="text">Language 5</li>
+          <li class="text">Language 6</li>
+          <li class="text">Language 7</li>
+        </ul>
+      </section>
+    </section>
+    <section class="middle-content">
+      <header class="middle-header">
+        <h2>BC</h2>
+      </header>
+      <div class="middle-img">
+        <img src="" alt= "Image 1">
+        <div class="divider-box1"></div>
+        <img src="" alt= "Image 2">
+        <div class="divider-box2"></div>
+        <img src="" alt="Image 3">
+      </div>
+
+    </section>
+    <section class="right-content">
+    <header class="right-header">
+      <h1>Campbell</h1>
+    </header>
+    <section class="about">
+      <h3>ABOUT</h3>
+      <p class="bold">FrontEnd Dev + Dreamer</p><br>
+      <p class="text">02/16/1991</p><br>
+      <p class="location">Location</p>
+    </section>
+    <section class="projects">
+      <h3>PROJECTS</h3>
+      <p class="bold">Year</p> <br>
+      <p class="text">Company- Job [explenation]</p> <br>
+      <p class="location">Location</p>
+      <p class="bold">Year</p> <br>
+      <p class="text">Company- Job [explenation]</p> <br>
+      <p class="location">Location</p>
+      <p class="bold">Year</p> <br>
+      <p class="text">Company- Job [explenation]</p> <br>
+      <p class="location">Location</p>
+      <p class="bold">Year</p> <br>
+      <p class="text">Company- Job [explenation]</p> <br>
+      <p class="location">Location</p>
+    </section>
+    <section class="intrests">
+      <h3>INTRESTS</h3>
```

Comment:
It's **_interests_** as a heads up; some employers care about spelling, although it doesn't matter too much.